cmd/podman/containers/create.go

```diff
@@ -388,10 +388,7 @@ func createPodIfNecessary(cmd *cobra.Command, s *specgen.SpecGenerator, netOpts
 	if err != nil {
 		return nil, err
 	}
-	imageName := config.DefaultInfraImage
-	podGen.InfraImage = imageName
-	podGen.InfraContainerSpec = specgen.NewSpecGenerator(imageName, false)
-	podGen.InfraContainerSpec.RawImageName = imageName
+	podGen.InfraContainerSpec = specgen.NewSpecGenerator("", false)
```

Comment:
will this populate the image name in the spec before passing it to the remote client if that path is called? Looking at `NewSpecGenerator` it looks like it will just pass the empty string

Reply:
Yes it should be emtry string, and allow the server side to decide, based on the contents of containers.conf.